CHANGELOG.md

```diff
@@ -94,6 +94,12 @@ configuration is as follows:
   If a node finds a proof invalid, it will report it to the network by
   creating a malfeasance proof. The malicious node will then be blacklisted by the network.
 
+* [#5586](https://github.com/spacemeshos/go-spacemesh/pull/5586)
+  Do not try to publish proofs for malicious ATXs during sync.
+
+  Publishing is blocked during sync because `Syncer::ListenToATXGossip()` returns false, and thus every malicious ATX being
```

Comment:
i think the accurate is Syncer.ListenToGossip()

```go
if newSyncer.ListenToGossip() {
			return nil
		}
```

Reply:
There is actually a separate `ListenToATXGossip` function: https://github.com/spacemeshos/go-spacemesh/blob/develop/node/node.go#L1050-L1055
It looks like a rudiment, though, b/c it's exactly same as `ListenToGossip`, so it's probably worth updating, but maybe as a separate PR

Reply:
sorry, you are right there was a separate condition for atx handler.
it is not identical to ListenToGossip, as ListenToATXGossip is enabled earlier, once atxs are downloaded.